externalPlugins/EX-PO007-NamingConventions.js

```diff
@@ -66,9 +66,17 @@ var plugin = {
 var onPolicy = function(policy, cb) {
```

Comment:
The problem of the "missing metadata" as noted in PR #273 is valid.  And also, this code has other problems which should also be fixed.